WordPress/Classes/Networking/PostServiceRemoteREST.m

```diff
@@ -143,6 +144,47 @@ - (void)deletePost:(RemotePost *)post
            }];
 }
 
+- (void)trashPost:(RemotePost *)post
+          forBlog:(Blog *)blog
+          success:(void (^)(RemotePost *))success
+          failure:(void (^)(NSError *))failure
+{
+    NSString *path = [NSString stringWithFormat:@"sites/%@/posts/%@/delete", blog.dotComID, post.postID];
```

Comment:
Same as above, if Blog and Post are both mandatory, can we add asserts?
(Sorry about being annoying!)